index.js

```diff
@@ -12,16 +12,17 @@ module.exports = function (gulp) {
   $.path = require('path');
   $.sassModuleImporter = require('sass-module-importer');
 
-  const tasks = [
+  let tasks = [
     'clean',
+    typeof(pkg.gulpPaths.fractalConfig) == 'undefined' ? false : 'fractal',
     'fractal',
     'images',
     'scripts',
     'styles',
-    // Tasks have to come last
     'build',
     'default',
   ];
+  tasks = _array.compact(tasks);
```

Comment:
Better use a native JS approach instead of introducing a new module.

```js
tasks = tasks.filter(Boolean);
```

Any particular case why you return `false` over an empty string?